interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -749,34 +706,11 @@ class="bold left text-uppercase text-top" style="display: flex;align-items:top">
             font-size:12px;
         }
 
-        .right {
-            text-align: right;
-            vertical-align: middle;}
-
-        .left {
-            vertical-align: middle;
-            text-align: left;
-        }
```

Comment:
Also for all the text-left and text-right stuff that I just said, also add the vertical-align-middle BS4 class